Sources/MessagingPush/MessagingPushConfigOptions.swift

```diff
@@ -1,21 +1,63 @@
+/**
+ Configuration options for push messaging module
+
+ Example usage:
+ ```
+ // create a new instance
+ let pushMessagingConfig = MessagingPushConfigOptions.Factory.create()
+ // now, it can be modified as
+ pushMessagingConfig.autoTrackPushEvents = false
+ ```
+ */
+// TODO: [CDP] Update docs to after module initialization changes for clarity on its usage
 public struct MessagingPushConfigOptions {
-    // Used to create new instance of MessagingPushConfigOptions when the MessagingPushAPN module is configured.
+    // Used to create new instance of MessagingPushConfigOptions when the MessagingPush module is configured.
     // Each property of the MessagingPushConfigOptions object can be modified by the user.
     public enum Factory {
         public static func create() -> MessagingPushConfigOptions {
             MessagingPushConfigOptions(
-                autoFetchDeviceToken: true
+                autoFetchDeviceToken: true,
+                autoTrackPushEvents: true,
+                autoTrackDeviceAttributes: true
             )
         }
     }
 
     public enum Keys: String { // Constants used to map each of the options in MessagingPushConfigOptions
         case autoFetchDeviceToken
+        case autoTrackPushEvents
+        case autoTrackDeviceAttributes
+    }
+    
+    mutating func apply(with dictionary: [String: Any]) {
```

Comment:
same comment about mutating function in the other config file.

Reply:
Responded [above](https://github.com/customerio/customerio-ios/pull/413#discussion_r1403258998)